### DISCUSSION TITLE
Release plan

### DISCUSSION THREAD

**Discussion by setiah** [Opster/opensearch-k8s-operator#92] [General]
Hi All, 

Great work going on here for kubernetes operator experience, kudos to everyone! Creating this discussion to work backwards on the release plan for the first OpenSearch k8s operator experience. I see there's a good [dev-plan](https://github.com/Opster/opensearch-k8s-operator/blob/main/docs/designs/dev-plan.md) but I couldn't find much on the release, hence created one. Please feel free to route me if there exists one or you think there's a better place to discuss this :) 

Questions:
1. Do we have a release qualifier for the first release to indicate the type of release? (Alpha/Beta/GA) 
2. What is our tentative release date?
3. What is the scope of features for the first release? ([dev-plan](https://github.com/Opster/opensearch-k8s-operator/blob/main/docs/designs/dev-plan.md) phase 1?) 
4. What are the current blockers? 

Appreciate your inputs, thanks!

**Comment by setiah** [comment]
For 1, in the last zoom meeting we talked about having a Beta release. Not sure if that's still the plan?

**Comment by segalziv** [comment]
Hi @setiah ,
Right, we had a small delay last week due to a blocker that was fixed, covid, etc. 
We plan to:
1. Release a beta release this week. We plan to get some feedback from different deployments and users, and we will release the GA is a week or two.
2. The beta tentative release date is 6th of April. The GA is planned for the 20th of April.
3. The scope of the GA is [dev-plan](https://github.com/Opster/opensearch-k8s-operator/blob/main/docs/designs/dev-plan.md) phase 1 indeed.
4. there was a blocker related to the security config last week, but that got solved. For now, there is no blocker. We are still planning to release to [operatorhub](https://operatorhub.io/), but that might be delayed due to the need for some adjustments. Other than that there are still a few open tasks in the dev plan, which most likely will be ready for the GA, but no blockers anyway.

**Reply by setiah** [reply]
Thanks for the answers @segalziv! Glad to hear there are no active blockers, and excited for the release.
